src/Common/nunit/CommandLineOptions.cs

```diff
@@ -40,7 +40,9 @@ namespace NUnit.Common
     public class CommandLineOptions : OptionSet
     {
         private bool validated;
+#if !NETCORE
```

Comment:
It would help to have a summary somewhere of what options are unavailable in NETCORE and why.